nginx.conf

```diff
@@ -33,13 +33,8 @@ http {
             add_header Cache-Control "private, max-age=0, no-cache";
             root   /nginx/html;
             index  index.html;
-        }
-        location ~* \.(?:ico|css|js|gif|jpe?g|png|woff|ttf)$ {
-        #--- Some basic cache-control for static files to be sent to the browser
-            expires max;
-            add_header Pragma public;
-            add_header Cache-Control "public, must-revalidate, proxy-revalidate";
-            root   /nginx/html;
+
+            include  /nginx/conf/mime.types;
```

Comment:
why was that changed?

Reply:
couldn't make first party work locally without it